conans/client/conan_api.py

```diff
@@ -626,6 +626,11 @@ def config_get(self, item):
             result = self.app.config.storage_path
         else:
             result = self.app.config.get_item(item)
+        if item is None:
+            # TODO: Provide a helper
+            result += "\n[conf]\n"
+            result += self.app.cache.new_config.dumps() + "\n"
```

Comment:
No, this is not the feature. It is not about listing current configuration.

It is about listing all possible configurations that are in our codebase, together with an explanation/documentation of what it does. Kind of ``conan --help`` for commands, but for configurations. Could be in ``conan config --help``, for example, or ``conan config list``.

At some point we might use it to validate user inputs, to avoid typos, for example.

Reply:
We don't populate new_config, we just read if a key is present or not. For example: https://github.com/conan-io/conan/blob/12600f3975704031782bfe0dd3c571a2ac36a05d/conans/client/conf/required_version.py#L33

We will to centralize them and populate empty values.

Reply:
Centralize their definition:
```python
confs = {
    "core.required_conan_version": "help for this config",
    "tools.microsoft.....": "help for this one too",
}
```